react-app/src/api/index.js

```diff
@@ -0,0 +1,86 @@
+import Account from 'account';
+import { createEntry } from 'utils/entry';
+
+const _HOST = 'http://localhost/';
+
+class ErrorCannotGetEntries extends Error {}
+
+class Api {
+  initAccount(username, keyPair) {
+    this._account = new Account(username, keyPair);
+  }
+
+  _getUrl(path, withToken = false) {
+    let url = `${_HOST}${path}`;
+    if (withToken) url += `?token=${this._token}`;
```

Comment:
Maybe always use the token if it's not null.

Reply:
Will this be always true for the server? If so, I think that it will be OK

Reply:
If you set the token when you sign in, and set it to null when you log out, then yes, everything in between will need a token.

Reply:
Done.